scripts/build.sh

```diff
@@ -2,9 +2,16 @@
 
 set -e
 
+declare -A EC_MODEL
+EC_MODEL["darp5"]="darp5"
```

Comment:
I don't like this being inside of build.sh. I recommend instead having a file, `ec.config`, in each model's directory, similar to `coreboot.config`. This file would have contents like this:
```
BOARD=system76/lemp9
```

If this file exists, and `ec.rom` does not exist, it would build System76 EC by copying that file to config.mk in the ec submodule.